README.md

```diff
@@ -41,6 +41,8 @@ The project is not ready for use. We hope to have full sync implemented sometime
 
 Rust minimum required version to build this project is 1.65.0 published 02.11.2022
 
+To test Reth, you will need to have [Geth and Lighthouse installed](https://github.com/paradigmxyz/reth/blob/main/book/installation.md).
```

Comment:
Wait we don't need light house do we? We should write a separate doc later about how to configure the CL, but for interop p2p tests it's just geth no?

Reply:
We don't and I don't think we should stick to Lighthouse for that section - all CLs work the same (mostly) in this regard. All of them also have their own docs on how to set this up, so my recommendation is that we (at some point later) just list out CL's we know work and link to their docs on setting it up as it might change, and I don't think we should be maintaing per-CL instructions on how to connect to execution clients